ai-machine-learning/thoth-s2i-advise/finish.md

```diff
@@ -0,0 +1,7 @@
+Thank you for trying to build usinng Thoth s2i build process. 
```

Comment:
maybe rephrase to: "Thank you for trying to create an application using the Thoth's s2i build process."?

Reply:
Thanks I have changed it :)